index.js

```diff
@@ -1,13 +1,15 @@
 const express = require('express')
-const bodyParser = require('body-parser')
 const movies = require('./movies')
 const { response } = require('express')
+const { getAllMovies, getMovieByTitle, getMovieByDirectors } = require('./controller/movies')
 
 const app = express()
 
-app.get('/', (request, response) => {
-  return response.send(movies)
-})
+app.get('/', getAllMovies)
+
+app.get('/:title', getMovieByTitle)
+
```

Comment:
The movie title and director search was supposed to be all in one function to execute.  You are missing some of the string, array methods to bring it out of the JSON file.  It only returns an empty array.